### DISCUSSION TITLE
An option to change the site's language

### DISCUSSION THREAD

**Discussion by Maximdraco** [RetroAchievements/RAWeb#1006] [Ideas]
An option that allows users to choose a preferred language so that everything on the page is translated into that language, I don't know if it would be feasible to use some AI to translate it.

Another option that occurs to me is that the devs create the translation for their achievements (name and description) and leaderboards, although a dedicated team could also be formed to help with that, translate the titles and descriptions of the achievements into different languages.

Those who are in charge of the site would have to translate the different texts on the page, I don't know how viable this function is in V1, maybe it will remain as something for V2.

**Comment by televandalist** [comment]
It's planned for V2 at least:

![Screenshot_20220528-112859](https://user-images.githubusercontent.com/45054151/170834208-68197946-da9c-4ba1-ae10-3f94eac56828.png)

With the way things have been going, maybe it could even be V1.94 😮

**Comment by Jamiras** [comment]
Done in #2767 (and many supporting PRs). Parts of site will honor the setting as they're converted.
